game/views.py

```
@@ -550,11 +553,32 @@ def verify_otp(request):
         return redirect('register')
 
     if request.method == 'POST':
+        
+        otp_created_at = request.session.get('otp_created_at')
+
+        if otp_created_at:
+            if time.time() - otp_created_at > 300:
```

Ruff (0.15.13)

[warning] Use a single `if` statement instead of nested `if` statements

Combine `if` statements using `and`

(SIM102)